pwndbg/commands/kversion.py

```diff
@@ -12,4 +12,4 @@
 @pwndbg.commands.OnlyWithKernelDebugSyms
 @pwndbg.commands.OnlyWhenPagingEnabled
 def kversion() -> None:
-    print(pwndbg.gdblib.kernel.kversion())
+    print(pwndbg.gdblib.kernel.kbanner())
```

Comment:
I think we can have a separate `kbanner` command that calls `kbanner`.

Reply:
i think i messed something up when renaming `kversion` to `kbanner` so i reverted that change and instead called the new function which returns the numeric tuple `krelease`